x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/pagerduty.tsx

```diff
@@ -74,6 +76,21 @@ export function getActionType(): ActionTypeModel {
           )
         );
       }
+      if (actionParams.timestamp && !hasMustacheTokens(actionParams.timestamp)) {
```

Comment:
This seems like it's parsing the action params to see if the date field doesn't have a mustache template, and then parsing it to see if it's a valid date.  That makes sense, but I wonder how often people will be setting literal dates in their action parameters like this.  Is it worth it?

Reply:
Well, we've already had a bug reported for not doing it so I get the feeling out UX isn't really good enough here.

If a specific format is required for this field, we should be able to validate that only content of that format is defined for it. The fact that we also want to support Mustache shouldn't mean the user can input bad data here... arguably we shouldn't allow any free values here at all, but rather we should only offer valid sources of Datetime strings, but when I looked into that I realised that was a huge change that didn't feel right as a patch bug fix. 🤷